src/cheri/riscv-priv-integration.adoc

```diff
@@ -480,31 +480,31 @@ Disabling CHERI register access prevents low-privileged {cheri_int_mode_name} so
 from interfering with the correct operation of higher-privileged {cheri_int_mode_name} software
 that do not perform <<ddc>> switches on trap entry and return.
 
-Disabling CHERI register access allows harts supporting CHERI to be fully
+Disabling {CRE} access allows harts supporting CHERI to be fully
 compatible with standard RISC-V, so CHERI instructions, such as <<CRAM>>, that
-do not change the state of CHERI CSRs raise exceptions when CRE=0.
+do not change the state of CHERI CSRs raise exceptions when {CRE}=0.
 This is the default behavior on reset.
 ====
 
 xref:cheri_behavior_cre_mode[xrefstyle=short] summarizes the behavior of a hart
-in connection with the <<section_cheri_disable,CRE>> and the <<cheri_execution_mode>> while in a privilege other than debug mode.
+in connection with the <<section_cheri_disable,{CRE}>> and the <<cheri_execution_mode>> while in a privilege other than debug mode.
 
-.Hart's behavior depending on the effective <<section_cheri_disable,CRE>> and <<cheri_execution_mode>>
+.Hart's behavior depending on the effective <<section_cheri_disable,{CRE}>> and <<cheri_execution_mode>>
```

Comment:
This title now reads a bit odd: `depending on the effective Y and CHERI Execution Mode`

Maybe something like "Y enable bits" instead of just {CRE}?

Reply:
I've updated following the rest of the text saying cheri enable/disable